keyboards/crkbd/keymaps/rjhilgefort/readme.md

```diff
@@ -0,0 +1,117 @@
+# [🐦 @rjhilgefort](https://twitter.com/rjhilgefort)'s Corne KBD (CRKBD) Layout
+
+My Corne KBD layout is inspired by [my Planck Layout](../../../planck/keymaps/rjhilgefort/readme.md), but with some additional features. Namely, this layout has Linux support.
+
+![](https://imgur.com/wp9ksTI.jpg)
+![](https://imgur.com/fnyiCnE.jpg)
```

Comment:
```suggestion
![](https://imgur.com/wp9ksTIl.jpg)
![](https://imgur.com/fnyiCnEl.jpg)
```

Reply:
Does the `l` fetch a different image? I'm unfamiliar with what this will do with `imgur`

Reply:
Ah, looks like it fetches a smaller image. Sounds good to me!

Reply:
Hmm, after I made this change, now the readme looks like this:

![image](https://user-images.githubusercontent.com/2034823/104045748-0204e780-51ad-11eb-8be5-9363951dfc49.png)

I think I prefer the larger image if that's okay.

Reply:
Here's what it looks like with the larger image:

![image](https://user-images.githubusercontent.com/2034823/104045944-5019eb00-51ad-11eb-874f-fa1ccce259ae.png)

Reply:
Those images are unnecessarily large, though.

Reply:
Does it hurt anything? They’re hosted on imgur so it’s not like it bloats the repo or anything. It’s a keymap for me and I like the large image in the readme? If others don’t like it they’ll skip past my keymap when exploring. I agree they’re larger than they need to be for the space though and wipe down when they load. Do you know how I can fetch a smaller image but one that still fills the space like the oversized one does? Similar to `l`, but a bit bigger?

Reply:
No, it doesn't bloat the repo, but for anyone viewing the image, the filesize is unnecessarily large. The original is about 3MB whereas the thumbnailed version is only 74kB.

Reply:
```suggestion
![](https://imgur.com/wp9ksTIh.jpg)
![](https://imgur.com/fnyiCnEh.jpg)
```

Reply:
Awesome, that's perfect! Thanks for the help with this!